README.md

```diff
@@ -89,3 +89,4 @@ This repository is designed to help newbies practice using Git and GitHub by cre
 - [John Doe](./john-doe/README.md)
 - [Sebastian Harvey](./jsstar/README.md)
 - [White Collin](./whitecollin/)
+- [naidendev](./naidendev/README.md)
```

Comment:
write your full name instead of nickname

Reply:
fixed it